weavedns/main.go

```diff
@@ -18,7 +18,8 @@ func main() {
 		justVersion bool
 		ifaceName   string
 		apiPath     string
-		dnsPort     int
+		udpPort     int
+		tcpPort     int
```

Comment:
According to the RFCs it is always 53.  We allow it to be changed for testing purposes, but one number should suffice for that.  I can't think of any other reason to allow two different numbers.